src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Validator.cs

```diff
@@ -87,10 +89,14 @@ protected virtual async Task<bool> ShouldRunAsync(ValidationContext context)
                 throw new TimestampComparisonException(timestampV2, timestampCatalog,
                     "The timestamp in the catalog is newer than the timestamp in the feed! This should never happen because all data flows from the feed into the catalog!");
             }
+            else if (timestampCatalog.Last == timestampV2.Last)
+            {
+                // If the timestamp metadata in the catalog is EQUAL to that of the feed, we are looking at the latest catalog entry that corresponds with this package, so run the test.
+                return ShouldRunTestResult.Yes;
+            }
 
-            // If the timestamp metadata in the catalog is LESS than that of the feed, we must not be looking at the latest entry that corresponds with this package, so skip the test for now.
-            // If the timestamp metadata in the catalog is EQUAL to that of the feed, we are looking at the latest catalog entry that corresponds with this package, so run the test.
-            return timestampCatalog.Last == timestampV2.Last;
+            // If the timestamp metadata in the catalog is LESS than that of the feed, we must not be looking at the latest entry that corresponds with this package, so we must attempt this test again later with more information.
+            return ShouldRunTestResult.Unknown;
```

Comment:
Instead of `ShouldRunTestResult.Unknown`, what do you think of `ShouldRunTestResult.RetryLater`? I think it makes the intent clearer

Reply:
I was actually thinking about this but forgot to update the PR. I like this much better.